PWGHF/HFC/Tasks/taskFlow.cxx

```diff
@@ -63,25 +65,68 @@ struct HfTaskFlow {
   Configurable<double> yCandMax{"yCandMax", -1., "max. cand. rapidity"};
   Configurable<std::vector<double>> binsPt{"binsPt", std::vector<double>{hf_cuts_d0_to_pi_k::vecBinsPt}, "pT bin limits"};
 
+  Service<o2::framework::O2DatabasePDG> pdg;
   HfHelper hfHelper;
   SliceCache cache;
 
+  // =========================
+  //      DATA : Filters & partitions
+  // =========================
+
   using FilteredCollisionsWSelMult = soa::Filtered<soa::Join<aod::Collisions, aod::EvSels, aod::Mults>>;
   using TracksWDcaSel = soa::Filtered<soa::Join<aod::TracksWDca, aod::TrackSelection>>;
   using HfCandidatesSel = soa::Filtered<soa::Join<aod::HfCand2Prong, aod::HfSelD0>>;
 
   //  Collision filters
   //  FIXME: The filter is applied also on the candidates! Beware!
   Filter collisionVtxZFilter = nabs(aod::collision::posZ) < zVertexMax;
-  //  Charged track filters
+
   Filter trackFilter = (nabs(aod::track::eta) < etaTrackAssocMax) &&
                        (aod::track::pt > ptTrackAssocMin) &&
                        requireGlobalTrackWoPtEtaInFilter();
+
   //  HF candidate filter
   //  TODO: use Partition instead of filter
-  Filter candidateFilter = aod::hf_sel_candidate_d0::isSelD0 >= selectionFlagD0 || aod::hf_sel_candidate_d0::isSelD0bar >= selectionFlagD0bar;
+  Filter candidateFilter = aod::hf_sel_candidate_d0::isSelD0 >= selectionFlagD0 ||
+                           aod::hf_sel_candidate_d0::isSelD0bar >= selectionFlagD0bar;
+
+  Preslice<aod::Tracks> dataPerCol = aod::track::collisionId;
+  // Do I have to adapt this preslice to MC ? How does it work exactly ?
+  // Preslice<aod::Tracks> mcRecPerCol = aod::track::collisionId;
+  Preslice<aod::McParticles> mcTruthPerCol = aod::mcparticle::mcCollisionId;
+
+  // =========================
+  //      MONTE-CARLO : Filters & partitions
+  // =========================
+
+  // Even add McCollisions in the join ?
+  // Kata adds subscribes to it but do not add it in the join
+  // using FilteredCollisionsWSelMultMC = soa::Filtered<soa::Join<aod::Collisions, aod::McCollisionLabels, aod::EvSels, aod::Mults, aod::McCollisions>>;
+  using FilteredCollisionsWSelMultMC = soa::Filtered<soa::Join<aod::Collisions, aod::McCollisionLabels, aod::EvSels, aod::Mults>>;
+  using TracksWDcaSelMC = soa::Filtered<soa::Join<aod::TracksWDca, aod::TrackSelection, aod::McTrackLabels>>;
+
+  // using aodCollisions = soa::Filtered<soa::Join<aod::Collisions, aod::EvSels, aod::Mults, aod::McCollisionLabels>>;
+
+  using aodMcCollisions = soa::Filtered<aod::McCollisions>;
+
+  using aodMcParticles = soa::Filtered<aod::McParticles>;
+
+  using aodTracks = soa::Filtered<soa::Join<aod::Tracks, aod::TracksDCA, aod::TrackSelection, aod::McTrackLabels>>;
```

Comment:
These names are too vague, misleading and not in line with the naming conventions.
Use names that clearly indicate what the table contains, e.g. `FilteredMcCollisions`, `FilteredMcParticles`, `FilteredTracksWDcaSelMcLabels`.

Reply:
I changed the names according to your proposition